fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
For consistency with the other fcrepo configuration elements, would you please change the following:
```
aws.accessKeyId  -> fcrepo.aws.accessKeyId
aws.secretKey  -> fcrepo.aws.secretKey
aws.bucket  -> fcrepo.aws.bucket
```
...for each of the three configuration files in this PR.